floodviz/static/js/floodviz_onready.js

```diff
@@ -26,20 +26,23 @@ document.addEventListener("DOMContentLoaded", function (event) {
 	var hydro_figure = FV.hydromodule(hydro_options);
 
 	// Use frames to link interactions
-	hydro_options['hover_in'] = map_figure.site_tooltip_show;
-	hydro_options['hover_out'] = map_figure.site_tooltip_remove;
-	hydro_options['click_on'] = map_figure.site_add_accent;
```

Comment:
Where do we use click_on? Shouldn't there just be a click interaction? Maybe we should talk about this.